server/app.py

```diff
@@ -290,6 +291,9 @@ async def validate_connection(request: ValidateConnectionRequest):
         if provider.lower() == "azureopenai":
             model = "azure/" + request.deployment_name
             model_extra = {"base_url": request.endpoint}
+        elif provider.lower() == "azureaifoundry":
```

Comment:
I thought this was already done in LSE ? passing in these params. do we not do it for validate connection, and only do it for inference runs?

Reply:
yep we only do it for inference runs, need to consolidate in LSE